analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -84,21 +84,53 @@ public void SimpleAssignment_ToLocalVariable_FromTrackedSymbol_VB()
             validator.ValidateTag("B", x => x.HasConstraint(DummyConstraint.Dummy).Should().BeTrue());
         }
 
+        [TestMethod]
```

Comment:
Let's merge these 4 into a `DataTestMethod`. They are all the same except the code and the second operation string.